run/tesla_api.py

```diff
@@ -143,16 +144,14 @@ def _get_api_token():
 
         return tesla_api_json['access_token']
 
-    # If the access token is not already stored in tesla_api_json AND the
-    # user didn't provide a token pair in the teslausb_setup_variables.conf,
-    # attempt to use the login credentials from teslausb_setup_variables.conf
-    # to create a new token pair, if they exist.
-    elif SETTINGS['tesla_email'] and SETTINGS['tesla_password']:
-        # Create a new pair of tokens from the OAuth API.
-        tesla = teslapy.Tesla(SETTINGS['tesla_email'], SETTINGS['tesla_password'])
-        tesla.fetch_token()
+    # If the access token is not already stored in tesla_api_json AND the user provided a refresh_token force it into the client to get a proper token
```

Comment:
break this up into two lines

Reply:
what's the maximum line length?

Reply:
Please keep it under 100 characters for readability on smaller screens

Reply:
done